src/modules/editor/components/ArticleFormBase.tsx

```diff
@@ -91,46 +95,64 @@ export class ArticleFormBase extends React.Component<IProps, IState> {
         )
     }
 
-    private onTitleChange = (title: string) => {
+    private handleTitleChange = (title: string) => {
         this.setState({
             title
         })
     }
 
-    private onVolumeChange = (volume: string) => {
+    private handleVolumeChange = (volume: string) => {
         this.setState({
             volume
         })
     }
 
-    private onIssueChange = (issue: string) => {
+    private handleIssueChange = (issue: string) => {
         this.setState({
             issue
         })
     }
 
-    private onSectionChange = (section: string) => {
+    private handleSectionChange = (section: string) => {
         this.setState({
             section
         })
     }
 
-    private onFocusChange = (focus: string) => {
+    private handleFocusChange = (focus: string) => {
         this.setState({
             focus
         })
     }
 
-    private onContributorsChange = (contributors: string[]) => {
+    private handleContributorsChange = (contributors: string[]) => {
         this.setState({
             contributors
         })
     }
 
-    private onEditorChange = (editorState: EditorState) => {
+    private handleEditorChange = (editorState: EditorState) => {
         this.setState({
             editorState
         })
     }
 
+    private handleMediumChange = (m: IMedium) => {
+        this.setState({
+            media: this.state.media.concat([m])
+        })
+    }
+}
+
+//used to move featured media to front of media list in sort
+//this means featured media will show up on front page first
+const sortByFeatured = (media: IMedium[]) => {
+    media.sort(sortByFeaturedSorter)
+}
+
+const sortByFeaturedSorter = (a: IMedium, b: IMedium) => {
```

Comment:
seems like the first function should be `sortMediaByFeatured` and the second `sortByFeatured`. In general I try to avoid naming functions as nouns.

Reply:
Agreed, changed